en/admin/migration.inc

```diff
@@ -166,6 +168,26 @@ The following is [options] used in **cubrid unloaddb**.
    
     This option makes it to unload the latest image of objects in data volumes when unloading instances (the MVCC version is ignored and it does not refer to the log volumes). It may include uncommitted data and deleted data that has not been vacuumed.
 
+.. option:: -t, --thread-count=COUNT
+
+    It is performed in parallel using as many threads as the number of COUNT, and COUNT must be in the range of 0 to 127.
+    If the setting is omitted, it is the same as specifying 1, and if it is specified as 0, it does not operate in thread mode.
+    Even if the thread mode is specified, if the table contains an Object type or a Set, MultiSet, or Sequence (List) type that contains an Object type, it will not operate in thread mode for that table.
```

Comment:
```suggestion
    Even if thread mode is specified, if a table contains an Object type or a Set, MultiSet, or Sequence (List) type that contains an Object type, it will not operate in thread mode for that table.
```